CLOUDP-272621: Update log downloads docs (#3253)
mongodb/mongodb-atlas-cli: docs/command/atlas-deployments-logs.txt

   * - --end
     - int
     - false

>      - UNIX Epoch-formatted ending date and time for the range of log messages to retrieve. This value defaults to the start date plus 24 hours if the start date is set. It defaults to the current timestamp if start date is not provided.

Reviewer: ```suggestion
     - Ending date and time for the range of log messages to retrieve, given in UNIX time. Defaults to the start date plus 24 hours, if the start date is set. If start date is not provided, ending time defaults to the current time.
```

Ditto for the other strings.
